### PR TITLE
Added tool for reset skyring password

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: root <root@dhcp42-78.lab.eng.blr.redhat.com>

### CODE REVIEW

File: src/reset_password.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+def connect_db(dbconfig):
+    client = pymongo.MongoClient(dbconfig['hostname'])
+    db = getattr(client, dbconfig['database'])
+    db.authenticate(dbconfig['user'], dbconfig['password'])
+    return db
+
+def main():
+    dbconfig = read_config_file()
+    db = connect_db(dbconfig)
+    password = getpass.getpass('Enter Password to reset:')
```
Comment (shtripat) [inline src/reset_password.py]: `Enter new password:`

Comment (GowthamShanmugam) [reply src/reset_password.py]: @shtripat i will change it

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat

**Comment by GowthamShanmugam** [comment]
please review

**Review by shtripat — COMMENTED** [review]
LGTM

**Review by cybernth — APPROVED** [review]

**Pull request merged**